solidity/test/bridge/Bridge.test.ts

```diff
@@ -4849,6 +4849,833 @@ describe("Bridge", () => {
+                  "0x0000000000000000000000000000000000000000"
+                )
+              })
+
+              it("should slash the wallet's operators", async () => {
+                // TODO: Add test when operator slashing is implemented
+              })
+
+              it("should emit RedemptionTimedOut event", async () => {
+                await expect(tx)
+                  .to.emit(bridge, "RedemptionTimedOut")
+                  .withArgs(
+                    data.wallet.pubKeyHash,
+                    data.redemptionRequests[0].redeemerOutputScript
+                  )
+              })
+            })
+          })
+
+          context("when the wallet is not the active wallet", () => {
+            const data: RedemptionTestData = SinglePendingRequestedRedemption
+            // Public key hash of a different wallet
+            const anotherWalletPublicKeyHash =
+              "0x123456789abcdef01234567891abcdef01234567"
+
+            before(async () => {
+              await createSnapshot()
+
+              await bridge.setWallet(data.wallet.pubKeyHash, {
+                ecdsaWalletID: data.wallet.ecdsaWalletID,
+                mainUtxoHash: ethers.constants.HashZero,
+                pendingRedemptionsValue: data.wallet.pendingRedemptionsValue,
+                createdAt: await lastBlockTime(),
+                moveFundsRequestedAt: 0,
+                state: walletState.Live,
+              })
+              await bridge.setWalletMainUtxo(
+                data.wallet.pubKeyHash,
+                data.mainUtxo
+              )
+              await bridge.setActiveWallet(anotherWalletPublicKeyHash)
+
+              const redeemerSigner = await impersonateAccount(
+                data.redemptionRequests[0].redeemer,
+                {
+                  from: governance,
+                  value: null, // use default value
+                }
+              )
+
+              await makeRedemptionAllowance(
+                redeemerSigner,
+                data.redemptionRequests[0].amount
+              )
+
+              await bridge
+                .connect(redeemerSigner)
+                .requestRedemption(
+                  data.wallet.pubKeyHash,
+                  data.mainUtxo,
+                  data.redemptionRequests[0].redeemerOutputScript,
+                  data.redemptionRequests[0].amount
+                )
+
+              await increaseTime(await bridge.redemptionTimeout())
+
+              await bridge
+                .connect(thirdParty)
+                .notifyRedemptionTimeout(
+                  data.wallet.pubKeyHash,
+                  data.redemptionRequests[0].redeemerOutputScript
+                )
+            })
+
+            after(async () => {
+              walletRegistry.closeWallet.reset()
+
+              await restoreSnapshot()
+            })
+
+            it("should not delete the active wallet public key hash", async () => {
```

Comment:
That makes sense to not duplicate all the assertions in this scenario but let's drop a comment about it.

Reply:
Done in a769909bce54058695ce018258de2ada00bb6175.